### ISSUE TITLE
Appraisal Sheet required

### ISSUE THREAD

**Issue by Olawale1** [frappe/erpnext#15002]
Hi,

The Appraisal system in ERPNext is in desperate need of a makeover. The current state may suffice for some businesses but the mid-sized and larger organizations I've encountered simply can't use it as it lacks even the basics of any real world appraisal template: 

1. Multiple Sections
2. Self Appraisal

Looking at the existing ERPNext structure, I think the quickest way to achieve multiple sections would be to create an 'Appraisal Sheet' document which will contain multiple Appraisal forms in a child table. The idea is to create an Appraisal form for each section required. So, for example, an organization that has 3 sections in their Appraisals for HR/Admin Assistants will make 3 appraisal templates (i.e. HR/Admin Assistant (Section 1), HR/Admin Assistant (Section 2), HR/Admin Assistant (Section 3)). The Employee would thus be required to fill each of these 3 Sections individually but the combined aggregate will be gotten by using an Appraisal Sheet

Basically, the Appraisal Sheet will be very similar to the Appraisal form with a few minor differences as seen below
![appraisal sheet](https://user-images.githubusercontent.com/9269337/43161613-77c618e6-8f80-11e8-8427-9ce7130b7ed1.png)

Clicking on the 'Fetch Appraisals' button will automatically populate the child table with the appraisal forms that have 'Start Date >= Appraisal Sheet Start Date' AND 'End Date <= Appraisal Sheet End Date'

A weightage should be set on each row in the child table with the total being 100 (just like we have in the Appraisal form) and the aggregate score is automatically updated accordingly

Thanks
